packaging/install/drain-cleaner/certmanager/041-Certificate.yaml

```diff
@@ -13,4 +13,6 @@ spec:
     - strimzi-drain-cleaner.strimzi-drain-cleaner
     - strimzi-drain-cleaner
   issuerRef:
+    group: cert-manager.io
+    kind: Issuer
```

Comment:
these are the two default values for this reason they weren't here. I would leave it as it is.

Reply:
It's in the install files, and it was fixed before, why it should be different?

Reply:
I'm just copying the stuff from the release itself to these folders. I would keep it consistent

Reply:
which install files? they are not here https://github.com/strimzi/strimzi-kafka-operator/blob/main/install/drain-cleaner/certmanager/041-Certificate.yaml or am I misleading what you mean.

Reply:
so did anything change from the previous release? before 1.3.0 these two fields weren't here.

Reply:
there it will be propagated with new release of Strimzi. In the packaging the release files of the Drain Cleaner are added.
https://github.com/strimzi/drain-cleaner/blame/main/packaging/install/certmanager/041-Certificate.yaml

Reply:
Ah ok got it ;-)